src/gatling/resources/bodies/prl/c100/PRLApplicantDetails.json

```diff
@@ -160,49 +160,49 @@
           "solicitorTelephone": null,
           "respondentLivedWithApplicant": null,
           "address": {
-            "AddressLine1": "8 Millwood Road",
-            "AddressLine2": null,
-            "AddressLine3": null,
-            "PostTown": "Hounslow",
-            "County": null,
-            "PostCode": "TW3 2HH",
+            "AddressLine1": "${addressLines(0)}",
+            "AddressLine2": "${addressLines(1)}",
+            "AddressLine3": "",
+            "PostTown": "${addressLines(2)}",
+            "County": "",
+            "PostCode": "${addressLines(3)}",
             "Country": "United Kingdom"
           },
           "solicitorOrg": {
-            "OrganisationID": "S46F334",
-            "OrganisationName": "YAyCkCHEPG"
+            "OrganisationID": "${orgref}",
+            "OrganisationName": "${orgname}"
           },
           "solicitorAddress": {
-            "AddressLine1": "8 Millwood Road",
-            "AddressLine2": null,
-            "AddressLine3": null,
-            "PostTown": "Hounslow",
-            "County": null,
-            "PostCode": "TW3 2HH",
+            "AddressLine1": "5 Perf Test Street",
+            "AddressLine2": "PerfTest",
+            "AddressLine3": "",
+            "PostTown": "PerfTest Town",
+            "County": "",
+            "PostCode": "PR1 1PRF",
             "Country": "United Kingdom"
           }
         },
-        "id": "71dcf128-41b5-4460-96bb-b337a4a41fe9"
+        "id": null
```

Comment:
I think we need to double check changing this to null, as it was previously a UUID in this field.